test/sharness/t0025-ctl-status-report-commands.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# MIT Licensed
+#
+
+test_description="Test ctl's status reporting functionality.  Test errors on incomplete commands"
+
+. lib/test-lib.sh
+
+test_expect_success "cluster-ctl version looks good" '
+    ipfs-cluster-ctl version >version.txt &&
+    egrep "[0-9]+\.[0-9]+\.[0-9]" version.txt >/dev/null &&
+    rm version.txt
+'
+
+test_expect_success "cluster-ctl can read id" '
+    ipfs-cluster-ctl id >id.txt &&
+    grep "> Addresses:" id.txt >/dev/null &&
```

Comment:
In this test im happy if `ipfs-cluster-ctl id | grep -q "id in the configuration"` succeeds.